terraform/modules/fhir/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "region" {
+  type        = string
+  description = "The region of the project and the related data sets"
+  default     = "us-west1"
+}
+
+variable "time_zone" {
+  type        = string
+  description = "The default timezone used by this dataset. Must be a either a valid IANA time zone name or empty, which defaults to UTC."
+  default     = "UTC"
+}
+
+variable "dataset_name" {
+  type        = string
+  description = "The resource name for the Dataset."
+  default     = "PHDI_DATASET"
+}
+
+variable "fhirstore_name" {
+  type        = string
+  description = "The resource name for the FhirStore."
+  default     = "PHDI_FHIRSTORE"
+}
+
+variable "fhir_version" {
+  type        = string
+  description = "The FHIR specification version. Default value is STU3. Possible values are DSTU2, STU3, and R4."
+  default     = "R4"
+}
```

Comment:
I'm curious about the choice here to pull these values out into a separate `variables.tf` file. I don't necessarily think this is wrong, but it is different from how we have set similar values elsewhere. For example, in the `storage` module we have set `location`, similar to `region` in this case, directly in `main.tf`. Generally, we have been creating variables like this when we need to pass values between modules. Again, I'm not convinced that this is wrong, but for the sake of consistency within the repo I think we should agree on an approach. I'll see if I can find any docs addressing best practice here. @nickclyde what do you think? 

My current thoughts on the merits of this approach are that the `description` parameter makes it very clear to a reader what the purpose of each value is, however simply defining the values in `main.tf` eliminates the need for this extra file and makes it easier to see the actual values.

Reply:
Thinking on this a little more and now I'm wondering if we should go with a third approach. What about defining these values (or at least some like `region` and `location` ) in `terraform/vars` instead of `terraform/modules`? This would give more granular control over the configuration of each environment we deploy to. For instance, we are using "us-west1" for Skylight, but it might make more sense to keep VA data in "us-east1". Looking forward to hearing both of your thoughts on this.

Reply:
Excellent points and I was thinking the same thing.  This was just to get a dataset and datastore created.  However, my original thought was to have all these parameters in a variable file at the terraform/vars/ level and one for each of the different environments that could be configured per client.